components/Footer.tsx

```diff
@@ -109,10 +109,10 @@ export const FooterImpl: React.FC = () => {
             <FaGithub />
           </a>
         )}
-        <a href="https://drive.google.com/file/d/1966dZX4w8KF84IpvKdBds5ILVYiu3N-k/view?usp=sharing" 
+{/*         <a href="https://drive.google.com/file/d/1966dZX4w8KF84IpvKdBds5ILVYiu3N-k/view?usp=sharing" 
```

Comment:
Commenting this out caused the entire production to break. Alternative action is to change the link to route to the other portfolio